apps/server/lib/lexical/server/code_intelligence/completion/translations/macro.ex

```diff
@@ -6,13 +6,62 @@ defmodule Lexical.Server.CodeIntelligence.Completion.Translations.Macro do
 
   use Translatable.Impl, for: Result.Macro
 
-  @snippet_macros ~w(def defp defmacro defmacrop defimpl defmodule defprotocol defguard defguardp defexception)
+  @snippet_macros ~w(def defp defmacro defmacrop defimpl defmodule defprotocol defguard defguardp defexception test)
 
   def translate(%Result.Macro{name: name}, _builder, _env)
       when name in ["__before_compile__", "__using__", "__after_compile__"] do
     :skip
   end
 
+  def translate(%Result.Macro{name: "test", arity: 1} = macro, builder, env) do
+    label = "#{macro.name} (Define an empty test/1)"
+
+    snippet = """
+    test "${0:name}"
+    """
+
+    builder.snippet(env, snippet,
+      detail: macro.spec,
+      kind: :class,
+      label: label,
+      sort_text: builder.boost(label, 10)
+    )
+  end
+
+  def translate(%Result.Macro{name: "test", arity: 2} = macro, builder, env) do
+    label = "#{macro.name} (Define test/2)"
+
+    snippet = """
+    test "${1:name}" do
+      $0
+    end
+    """
+
+    builder.snippet(env, snippet,
+      detail: macro.spec,
+      kind: :class,
+      label: label,
+      sort_text: builder.boost(label, 10)
```

Comment:
I copy/pasted these from another test but these properties are not clear to me:
`kind` and `sort_text`

Reply:
They're part of the LSP spec. `kind` is the type of completion you're completing, for example: `:function` or `:macro`. Sort text is what it sounds like, text that's used when ordering results by the editor.